parkhangs-webapp/src/features/parks/parksSlice.js

```diff
@@ -4,7 +4,12 @@ import { createSlice } from '@reduxjs/toolkit'
 const parksSlice = createSlice({
     name: 'parks',
 
-    initialState: [],
+    initialState: [
+        ["Dude Chilling Park", "2020-07-16T19:20-07:00"],
```

Comment:
maybe instead of having an event be an array, make it an object? like
 ```
{
  parkName: "Dude Chilling Park",
  eventTime: 1592508391, 
}
```

this will make it easier to use the data by keying into the object rather than guessing the location of certain attributes in the array.

Reply:
Yeah I was considering that, but I also recall us mentioning Event being array within array of parks. I can make the change now.

Reply:
yeah i think were were talking about each park having an array of event objects

Reply:
Oh, understood, I'll alter it as such then. Like this?
```
[
parkName: "Dude Chilling Park",
events: [
{
parkName: "Dude Chilling Park",
eventTime: 1592508391 //event 1
},
{
parkName: "Sunset Beach Park",
eventTime: 1892508391 //event 2
},
]
]
```

Reply:
or this:
`{
    parkName: "Dude Chilling Park",
         events: [
                {parkName: "Dude Chilling Park", eventTime: "2020-07-16T19:20-07:00"}
         ]
  }  
`